cmd/api-linter/integration_test.go

```diff
@@ -119,12 +127,14 @@ func TestRules_DisabledByInlineComments(t *testing.T) {
 	`
 
 	for _, test := range testCases {
-		disableInline := fmt.Sprintf("// (-- api-linter: %s=disabled --)\n", test.rule)
-		proto := strings.Replace(test.proto, "disable-me-here", disableInline, -1)
-		result := runLinter(t, proto, config)
-		if strings.Contains(result, test.rule) {
-			t.Errorf("Rule %q should be disabled by in-line comments", test.rule)
-		}
+		t.Run(test.testName, func(t *testing.T) {
+			disableInline := fmt.Sprintf("(-- api-linter: %s=disabled --)", test.rule)
```

Comment:
Remove "(--" and "--)".

Reply:
The common use case will include these tokens.